live-build/config/hooks/configuration/81-upgrade-repository.binary

```diff
@@ -80,6 +80,20 @@ EOF
 #
 rename 's/\%3a/:/g' binary/packages/*.deb
 
+#
+# Copy over the generated hotfix metadata.
+#
+if [[ -z "$HOTFIX_METDATA" ]]; then
+	touch hotfix_metadata
+else
+	aws s3 cp --only-show-errors "$HOTFIX_METDATA" hotfix_metadata
+fi
+
+if [[ ! -f "hotfix_metadata" ]]; then
```

Comment:
is this check necessary? e.g. I'm wondering that if the file doesn't exist, would the package fail to build? i.e. since the package requires that file, which wouldn't exist.. I'm not necessarily against an explicit check, it just seems a bit redundant, and inconsistent with other files required by the package (e.g. we don't do this for `packages.list`)..